### PR TITLE
Relax max_requests

### PR DESCRIPTION
**Pull request by dralley**
The root cause of the memory leak has been established and resolved, so these aggressive values are no longer necessary

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -243,8 +243,8 @@
   Integer[0] $api_service_worker_count = min(4, $facts['processors']['count']) + 1,
   Integer[0] $content_service_worker_timeout = 90,
   Integer[0] $api_service_worker_timeout = 90,
-  Integer[0] $api_service_worker_max_requests = 50,
-  Integer[0] $api_service_worker_max_requests_jitter = 30,
+  Integer[0] $api_service_worker_max_requests = 800,
+  Integer[0] $api_service_worker_max_requests_jitter = 100,
```
Comment (dralley) [inline manifests/init.pp:247]: These seem reasonable to me, but they're just spitballs.  Probably 4,000 and 500 would be fine too.

**Comment by ekohl** [comment]
> The root cause of the memory leak has been established and resolved, so these aggressive values are no longer necessary

Out of interest: can you share a link?

**Comment by dralley** [comment]
https://github.com/pulp/pulpcore/pull/4090

**Comment by dralley** [comment]
I believe the tests are testing changing those values rather than the current values, so they don't need to be changed.  Unless you mean something else.

**Comment by dralley** [comment]
@ekohl Pushed

**Comment by ekohl** [comment]
https://ci.theforeman.org/job/pulpcore-3.22-rpm-pipeline/ is still blocked (there was some issue in CentOS CI) and I'd like to wait for that to pass.

**Review by ekohl — APPROVED** [review]

**Pull request merged**